src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/dto/IepSummary.kt

```diff
@@ -39,21 +40,24 @@ data class IepSummary(
       return Duration.between(iepDate.atStartOfDay(), currentIepDate).toDays().toInt()
     }
 
-  fun daysOnLevel(): Int {
-    val currentIepDate = LocalDate.now().atStartOfDay()
-    var daysOnLevel = 0
+  val daysOnLevel: Int
+    get() {
+      if (iepDetails.isEmpty()) throw ValidationException("iepDetails not available")
 
-    run iepCheck@{
-      iepDetails.forEach {
-        if (it.iepLevel != iepLevel) {
-          return@iepCheck
+      val currentIepDate = LocalDate.now().atStartOfDay()
```

Comment:
I know this is the existing code wrapped in a getter...but similar to other comment, I think this "variable" should be called `today` to avoid any confusion ☺️